packages/mux-player/src/index.ts

```diff
@@ -593,12 +593,12 @@ class MuxPlayerElement extends VideoApiElement {
   /**
    * @deprecated please use ._hls instead
    */
-  get hls() {
+  get hls(): PlaybackEngine | undefined {
```

Comment:
thought(non-blocking): this should be inferrable if `this._hls` is typed as expected, but should not be a blocker.

Reply:
That's what I thought as well, but when I went to do this update, tsc was complaining that these weren't typed.